devbox.json

```diff
@@ -1,43 +1,44 @@
 {
-  "packages": [
```

Comment:
why got your change overwritten? 👀

Reply:
Ahhh, never mind, it has never been merged :D 

https://github.com/cloudfoundry/app-autoscaler-release/pull/2943/files#diff-3d4c81b2bef90b8980e7c7c4900d75007f13acf07d9e62aa509f2d53f9fa4ef9

Reply:
Yes, I wanted to pull it out from the docker image PR, as it can be useful for the MTA dev work